test/aria/utils/Html.js

```diff
@@ -51,6 +51,14 @@ Aria.classDefinition({
                                 this.assertEquals(got, value, "data-" + dataKey + " should be %2, got %1");
                             }
                         }
+                    } else if (key === "aria") {
+                        for (var ariaKey in attribute) {
+                            if (attribute.hasOwnProperty(ariaKey)) {
+                                var value = stringUtil.encodeForQuotedHTMLAttribute(attribute[ariaKey]);
```

Comment:
There should be no call to `encodeForQuotedHTMLAttribute` here (it was also wrong line 46), but, anyway, the test does not check attributes containing quotes, so the encoded version is identical to the non-encoded version.